extensions/amp-form/0.1/utils.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks if a form field is in its default state.
+ * @param {!Element} field
+ * @return {boolean}
+ */
+export function isFieldDefault(field) {
```

Comment:
nit: Let's put this method in `src/form.js` with the other form utility methods

Reply:
For sure! Mind if I do that after #22536 is merged? Not sure if there could be merge conflicts.